### ISSUE TITLE
[RFC] Download as PDF

### ISSUE THREAD

**Issue by vrom911** [kowainik/kowainik.github.io#102]
Removing unnecessary layout for the PDF document and styling it differently.

I propose having a button on the side menu zone for each post.

**Comment by chshersh** [comment]
@vrom911 This is an excellent idea! It would be great to be able to download blog posts as PDF so you can read them offline :slightly_smiling_face: Maybe this can be done easily with `pandoc` :thinking:

**Comment by vrom911** [comment]
Yeah, I also was thinking about reusing pandoc itself, need to investigate further!
